qiskit_research/utils/gates.py

```diff
@@ -136,6 +136,41 @@ def __array__(self, dtype=None):
         return np.array([[0, -1j], [1j, 0]], dtype=dtype)
 
 
+class PiPhiGate(Gate):
```

Comment:
The name is not terribly descriptive. I wonder if we can come up with a better name. How about `PhasedXGate`?

Reply:
Sorry but it's already used in the [DD literature](http://arxiv.org/abs/2207.03670).